search-service/src/main/kotlin/com/egm/stellio/search/entity/web/BatchAPIResponses.kt

```diff
@@ -39,7 +40,7 @@ data class BatchOperationResult(
     fun addEither(either: Either<APIException, *>, entityId: URI, csrId: URI? = null) {
         either.fold(
             { this.errors.add(BatchEntityError(entityId, it.toProblemDetail(), csrId)) },
-            { this.success.add(BatchEntitySuccess(csrId ?: entityId)) }
+            { this.success.add(BatchEntitySuccess(csrId ?: NGSILD_NULL.toUri())) }
```

Comment:
or `NGSILD_LOCAL`? it exists in `JsonLdUtils` but it is not used. I prefer local over null anyway.